front/src/app/LogsTable.tsx

```diff
@@ -0,0 +1,39 @@
+
+type Props = {
+    logs: {
+        id: number;
+        datetime_utc: number;
+        input_string: string;
+        output_string: string;
+        total_tokens: number;
+    }[]
+}
```

Comment:
Have you looked into using the prisma model here?

Reply:
good shout!